<launch>
    <node name="dynamic_acknowledgement" pkg= "hera_speech" type="dynamic_responses.py" output="screen">
    </node>
    <node name="whisper" pkg= "hera_speech" type="master_whisper.py" output="screen">
    </node>
    <node name="answer" pkg= "hera_speech" type="new_answer.py" output="screen">
    </node>
    <node name="keywords" pkg= "hera_speech" type="new_keywords.py" output="screen">
    </node>
    <!-- <node name="move" pkg= "hera_speech" type="new_move.py" output="screen">
    </node> -->
    <node name="speak" pkg= "hera_speech" type="speak.py" output="screen">
    </node>
    <node name="task" pkg= "hera_speech" type="task_initializer.py" output="screen">
    </node>
        
</launch>
